Gruntfile.js

```diff
@@ -371,6 +372,10 @@ module.exports = function(grunt) {
 			docs_dist: {
 				command: 'cd docs && grunt dist',
 				stdout: true
+			},
+			docs_grunt: {
+				command: 'cd docs && grunt',
+				stdout: true
```

Comment:
can you try to write `stdout: 'inherit'` to get all the colors back?

https://github.com/jharding/grunt-exec/issues/59